examples/nextjs-with-typescript/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
+    "dev:secure": "next dev --experimental-https",
```

Comment:
NOTE: This is needed for some DRM testing permutations (e.g. testing on something other than localhost, such as a mobile device locally)

Reply:
For more, see: https://nextjs.org/docs/app/api-reference/next-cli#https-for-local-development